creacion de fragmentos

infolimbert opened the pull request:
se cambio la forma de implementar los fragmentos usando mobie-navigation

infolimbert committed:
se implemento los fragmentos de una manera mas corta usando mobile-navigation

infolimbert committed:
pruebita para saber si se actualiza el PR

Amancaya commented on app/build.gradle.kts:
```diff
@@ -1,3 +1,5 @@
+
+
```
Eliminar estos espacios innecesarios

Amancaya commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -37,13 +37,31 @@
-        app:layout_constraintEnd_toEndOf="parent"/>
+        app:layout_constraintEnd_toEndOf="parent"
+        app:defaultNavHost="true"
+        app:navgraph="@navigation/mobile_navigation"/>-->
```
No olvides eliminar el codigo comentado

Amancaya commented on build.gradle.kts:
```diff
@@ -1,5 +1,7 @@
+
+
```
Eliminar estos espacios innecesarios

infolimbert committed:
se limpio el codigo comentado, los espacios. Ahora se implemento un appBarConguration para que sea dinamico el titulo, y se disenio el layout del fragmento LocationFragment

infolimbert committed:
se crearon los permisos de ubicacion en el LocationFragment

infolimbert committed:
se crearon los permisos de ubicacion en el LocationFragment

infolimbert committed:
se anadio el tvLocation para poner la latitud y longitud
implementa el servicio de ubicación en primer plano (startForegroundService).
Configura FusedLocationProvider para alta precisión.
Establece la comunicación de ubicación mediante LocalBroadcastManager.

infolimbert committed:
Implementar rastreo de ubicación y geocodificación de dirección

Añade la funcionalidad completa de rastreo:
- Configura LocationService para correr como Foreground Service.
- Implementa el BroadcastReceiver en LocationFragment.
- Muestra la dirección postal obtenida por Geocoder en el TextView.
- Optimiza LocationRequest a 5 segundos con alta precisión.

infolimbert committed:
se anadio la identificacion de cruces de calle. tambien se puso como referencia si es que hay los POIs.

infolimbert committed:
se soliono el problema de que no habia un parametro para controlar la conexion de internet, necesario para el geocoder, ahora ya tenemos una notificacion

infolimbert committed:
se soluciono el problema de la notificacion que no se cerraba cuando cerraban la app con las tareas flotantes, ahora si se cierra la notificacion
